timesketch/lib/datastores/elastic.py

```diff
@@ -870,6 +884,29 @@ def flush_queued_events(self):
         self.import_events = []
         return return_dict
 
+    def get_field_mapping(self, index, fields):
+        """Get Elasticsearch Field Mapping
+
+        Returns:
+            ElasticSearch Field Mapping as json object.
+        """
+        return self.client.indices.get_field_mapping(index=index,
+                                                     fields=fields)
+
+    def aggregation_shim(self, index, aggregation_spec):
+        """Shim ahead of moving the Helper method 'elastic_aggregation'
```

Comment:
ok, so two things

a) can you explain what you mean by this being a `shim ahead of moving the helper...`

b) can you change the doc string here... if you look at the style guide (https://google.github.io/styleguide/pyguide.html#381-docstrings) a function docstring should be:

```
A docstring should give enough information to write a call to the function without reading the function’s code. The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable."""), except
```

You can have this line as the line after that.

def agg....
   ""Returns Elastic aggregation results .....

   Shim ahead of moving....

   Args: 
 ....
"""

Reply:
Looking at this PR, this method doesn't add anything. You can simplify it by removing this method and call `self.client.search()` as suggested earlier in this file.

Or am I missing something here? :)

Reply:
Will Update Docstring as above. @berggren I think i've replied to the wrong comment - the reasoning for the shim is here: https://github.com/google/timesketch/pull/1503#discussion_r536218345 

Though more than happy to change :D